.pylintrc

```diff
@@ -172,8 +173,7 @@ spelling-store-unknown-words=no
 [MISCELLANEOUS]
 
 # List of note tags to take in consideration, separated by a comma.
-notes=FIXME,
-      XXX,
+notes=XXX,
```

Comment:
Just wondering, why remove the FIXME from the notes that pylint warns about?

Reply:
I've removed this as there are a few `FIXME` notes around in the codebase that do need addressing. They should really be written up into issues, but would likely get overlooked as specific tasks as other work takes priority but by leaving them in place they act as an _aide-mémoire_ when myself or anyone else is looking at/working on that section of code.